raw_linux.go

```diff
@@ -223,19 +189,25 @@ func (p *packetConn) LocalAddr() net.Addr {
 
 // SetDeadline implements the net.PacketConn.SetDeadline method.
 func (p *packetConn) SetDeadline(t time.Time) error {
-	return p.SetReadDeadline(t)
+	if !p.noTimeouts {
```

Comment:
This configurable should go away because all platforms should now be able to use the runtime network poller.  Also remove from https://godoc.org/github.com/mdlayher/raw#Config.

Reply:
I'll open a separate issue for BSD support and see if I can find volunteers. But let's get rid of this tunable!